modules/client-configuration/pages/registration-webui.adoc

```diff
@@ -3,8 +3,11 @@
 
 Registering clients with the {productname} {webui} works for Salt clients only.
 
+// FIXME: change Salt bundle
 If you are bootstrapping Salt clients using the {webui}, you need to ensure that the client system has Python installed before you begin.
 For Salt clients running {sles}{nbsp}12 or older, you also need the [package]``python-xml`` package.
+If you use the Salt bundle, which now is the default, the needed Python environment will be provided automatically.
```

Comment:
Actually Python is still required to be installed on the system to bootstrap with web UI (bootstrapping with bootstrap script is not python dependent). But we will change it soon with implementing Salt-SSH with Salt Bundle

Reply:
ok change accordingly.